packages/cosmos-bin/src/main.rs

```diff
@@ -57,6 +57,10 @@ struct Opt {
         env = "COSMOS_GAS_MULTIPLIER"
     )]
     gas_multiplier: Option<f64>,
+
+    /// Referer header
+    #[clap(long, short, global = true)]
```

Comment:
I think it could be convenient to have an env var for this too.

Reply:
Pushed a fix to recognize environment variable.